Makefile

```
@@ -14,6 +18,14 @@ add-license-headers: $(GO_ADD_LICENSE)
 build:
 	@./hack/build.sh
 
+.PHONY: docker-build
+docker-build:
+	@docker buildx build --platform=$(PLATFORM) -t $(IMG) -f Dockerfile --rm .
+
+.PHONY: docker-push
+docker-push:
+	@docker push $(IMG)
```

```suggestion
docker-push:
	@if ! docker images $(IMAGE_REPOSITORY) | awk '{ print $$2 }' | grep -q -F $(IMAGE_TAG); then echo "$(IMAGE_REPOSITORY) version $(IMAGE_TAG) is not yet built. Please run 'make docker-image'"; false; fi
	@docker push $(IMG)
```
Can you add this line as well just like etcd-backup-restore: https://github.com/gardener/etcd-backup-restore/blob/master/Makefile#L124

---

Done